### PR TITLE
Findorigin

### PR DESCRIPTION
**Pull request by nienna73**
## What's in this PR:
* when the word already exists in MD, also save the translation (not just the origin)

### CODE REVIEW

File: validation/management/commands/findorigin.py [hunk 1]
```diff
@@ -33,27 +33,27 @@ def handle(self, *args, **options):
...
-            if not phrase.origin:
-                normalized = normalize_sro(phrase.transcription)
-                normalized = remove_diacritics(normalized)
-                if normalized in md_words:
-                    phrase.origin = "MD"
-                else:
-                    phrase.origin = "new"
-                phrase.save()
+            normalized = normalize_sro(phrase.transcription)
+            normalized = remove_diacritics(normalized)
```
Comment (eddieantonio) [inline validation/management/commands/findorigin.py:37]: Is it for sure that a term in the Maskwacîs dictionary will be written in normalized SRO, without diacritics?

File: validation/management/commands/findorigin.py [hunk 2]
```diff
@@ -33,27 +33,27 @@ def handle(self, *args, **options):
...
+    :return: a dict of Cree words and their translations
     """
-    words = []
+    words = {}
     with open("private/treatedMD.csv", "r") as f:
         reader = csv.DictReader(f)
 
         for row in reader:
-            words.append(row["SRO"])
+            words[row["SRO"]] = row["MeaningInEnglish"]
```
Comment (eddieantonio) [inline validation/management/commands/findorigin.py:56]: I'm not sure what file `treatedMD.csv` is, but are we sure that each `row["SRO"]` is unique? What happens if multiple rows have the same `row["SRO"]`?

**Review by eddieantonio — APPROVED** [review]
LGTM\*, provided that the questions I asked are non-issues.

File: validation/management/commands/findorigin.py (hunk 2 above)
Comment (eddieantonio) [inline validation/management/commands/findorigin.py:56]: I guess to answer my own question, you could have this:

```suggestion
            assert row["SRO"] == remove_diacritics(normalize_sro(row["SRO"]))
            words[row["SRO"]] = row["MeaningInEnglish"]
```

Or, if this is okay:

```suggestion
            normalized = remove_diacritics(normalize_sro(row["SRO"]))
            words[normalized] = row["MeaningInEnglish"]
```

File: validation/management/commands/findorigin.py (hunk 1 above)
Comment (nienna73) [reply validation/management/commands/findorigin.py:37]: The words in the Maskwacîs dictionary have been pre-treated to not contain diacritics to make matching easier. I understand this means I could run into some homograph ambiguities, but I think those can be manually resolved in the interface.

File: validation/management/commands/findorigin.py (hunk 2 above)
Comment (nienna73) [reply validation/management/commands/findorigin.py:56]: I'm 95% confident that each `row["SRO"]` is unique, and if it isn't then the later translation will overwrite any earlier ones

**Pull request merged**